ntp-ntpv5.xml

```diff
@@ -806,7 +837,7 @@
           frequency of the clock has to be adjusted to correct time errors that
           accumulate due to the frequency error (e.g. caused by changes in the
           temperature of the crystal). Faster corrections of time can minimize
-          the time error, but increase the frequency error, which transfers to
+          the time error, but in some implementations it can increase the frequency error, which transfers to
```

Comment:
In which implementations it doesn't impact frequency error? I think any phase correction creates a frequency error. If it's sufficiently slow, it would be hidden in the instability of the time source and local clock, but it's still there, right?

Reply:
The point was that this is implementation specific. For example, an implementation could update the frequency and never update the time. Anyway, it is just a side note and if you feel that it does not improve the coherency it can be removed.